Token.sol

```diff
@@ -34,6 +34,12 @@ contract TokenInterface {
     mapping (address => uint256) balances;
     mapping (address => mapping (address => uint256)) allowed;
 
+    /// Public variables of the token, all used for display 
```

Comment:
none of those variables are ever set, so they should be part of the constructor.